src/util.ts

```diff
@@ -7,15 +6,16 @@ import projectCacheDir from 'find-cache-dir';
 import findUp from 'find-up';
 import fs from 'fs';
 import got, {CancelableRequest, Response} from 'got';
+import mkdirp from 'mkdirp';
 import open from 'open';
 import path from 'path';
-import {SnowpackConfig, BuildScript} from './config';
-import mkdirp from 'mkdirp';
+import rimraf from 'rimraf';
+import {SnowpackConfig} from './config';
 
 export const PIKA_CDN = `https://cdn.pika.dev`;
 export const GLOBAL_CACHE_DIR = globalCacheDir('snowpack');
 export const RESOURCE_CACHE = path.join(GLOBAL_CACHE_DIR, 'pkg-cache-1.4');
```

Comment:
Just curious, what the 1.4 and 2.6 are in this case? Should they be incrementing with our package version?

Reply:
Good call, will add a comment. Right now we only increment these when something about the data structure has changed. This shows that 1.4 was the last time the data structure changed, so we've been able to reuse the same cache since then.